app/components/cards/collectionCard.tsx

```diff
@@ -0,0 +1,149 @@
+"use client";
+import React from "react";
+import {
+  Card,
+  CardHeading,
+  Text,
+  CardContent,
+  Tooltip,
+  Link,
+} from "@nypl/design-system-react-components";
+import styles from "../swimlanes/Swimlanes.module.css";
+import { CollectionCardData } from "../..//types/CollectionCard";
+interface CollectionCardProps {
+  slug: string;
+  index: number;
+  isLargerThanLargeTablet: boolean;
+  collection: CollectionCardData;
+}
+
+const CollectionCard = ({
+  slug,
+  index,
+  isLargerThanLargeTablet,
+  collection,
+}: CollectionCardProps) => {
+  return (
+    <Card
+      sx={{ display: "grid" }}
+      id={`card-${slug}-${index}`}
+      mainActionLink={collection.url}
+      imageProps={{
+        alt: "",
+        id: `image-${slug}-${index}`,
+        isLazy: true,
+        aspectRatio: "twoByOne",
+        src: collection.imageURL,
+      }}
+    >
+      <CardHeading
+        id={`row-card-heading-${slug}-${index}`}
+        level="h3"
+        size="heading5"
+        className={styles.collectiontitle}
+        noOfLines={3}
+      >
+        {isLargerThanLargeTablet ? (
+          <Tooltip content={collection.title}>
+            <Link href={collection.url} sx={{ marginBottom: "0" }}>
+              {collection.title}
+            </Link>
+          </Tooltip>
+        ) : (
+          collection.title
+        )}
+      </CardHeading>
+      <CardContent>
+        <Text
+          id={`item-count-${slug}-${index}`}
+          size="subtitle2"
+          fontWeight="medium"
+          __css={{
+            display: "none",
+            [`@media screen and (min-width: 400px)`]: {
```

Comment:
Use the breakpoint values found in `app/utils/breakpoints.ts`. In this case I think this should be `lgMobile`.

Reply:
The bigger problem is that `CollectionCard` is composed to replaced the `Card` component in `swimLanes.tsx` and so far it's coming along well. But when you get to replacing `Card` in `exploreFurther.tsx` with `CollectionCard`, it won't be a direct swap because the heading and content are _slightly_ different.

We can chat about some approaches tomorrow.

Reply:
I don't think CollectionCard should be used to replace the Card component in exploreFurther - I wasn't planning on swapping them out either but if we want to make a new ticket for that we can.

Reply:
I think CollectionCard should represent a Collection - the Explore Further section does not display Collections